src/lib/index.js

```diff
@@ -13,6 +13,8 @@ var d3 = require('d3');
 
 var lib = module.exports = {};
 
+require('./request_animation_frame');
```

Comment:
Since `raf` is already pretty well supported (ref http://caniuse.com/#feat=requestanimationframe), I'd vote for not including the polyfill in the plotly.js bundle by default.

I'd would move the `request_animation_frame.js` to [`dist/extras`](https://github.com/plotly/plotly.js/tree/master/dist/extras) and add a line in the `dist/` README.md generated [here](https://github.com/plotly/plotly.js/blob/f07a274fe7c215c06d3827f2a589f17421f72ce1/tasks/stats.js#L33).